src/components/10-atoms/checkbox/react/DemoUpdateLabelChildren.jsx

```diff
@@ -0,0 +1,21 @@
+import React, { useState, useEffect } from 'react';
+import AXACheckbox from './AXACheckboxReact';
+
+export default function App() {
+  const [label, setLabel] = useState("AAAAAA");
```

Comment:
What about "Meaningful Testdata"? :D 🗡

Reply:
Is the example our customer provided, but i can make it better on our example